.goreleaser.yml

```diff
@@ -122,6 +133,9 @@ dockers:
       - "{{ .Env.DOCKER_REPO }}armada:{{ .Major }}.{{ .Minor }}.{{ .Patch }}"
       - "{{ .Env.DOCKER_REPO }}armada:latest"
     build_flag_templates:
+      - --builder={{ .Env.DOCKER_BUILDX_BUILDER }}
```

Comment:
does this mean we will need to set these if using goreleaser locally?

Reply:
No, using it locally will use the default runner.